markets/bfp-market/test/integration/modules/PerpAccountModule.mergeAccounts.test.ts

```diff
@@ -568,6 +617,12 @@ describe('PerpAccountModule mergeAccounts', () => {
     assertBn.gt(fromDigestBefore.collateralUsd, 0);
     assertBn.gt(toDigestBefore.collateralUsd, 0);
 
+    // Assert that from account has at least two collaterals
+    const numberOfCollateralsWithBalance = fromDigestBefore.depositedCollaterals.filter((x) =>
+      x.available.gt(0)
+    ).length;
+
+    assertBn.gt(bn(numberOfCollateralsWithBalance), 1);
```

Comment:
Why can't this be `assertBn.equal(bn(numberOfCollateralsWithBalance), 2);`?

Reply:
Yea it can!